examples/zkdetect/main.go

```diff
@@ -0,0 +1,53 @@
+package main
```

Comment:
if it's not easy to remember/invoke, then I (and probably most others) won't actually do it. building the zkdetect binary provides a simple entrypoint for testing and doesn't rely on a dev having to remember the specific `go test` syntax. if you really don't want the binary, then I'd suggest wrapping the go incantation in a shell script so that it's easy to invoke